metrics/report/README.md

```diff
@@ -0,0 +1,61 @@
+# Kata Containers metrics report generator
+
+The files within this directory can be used to generate a 'metrics report'
+for Kata Containers.
+
+The primary workflow consists of two stages:
+
+1) Run the provided report metrics data gathering scripts on the system(s) you wish
+to analyse
+2) Run the provided report generation script to analyse the data and generate a
+report file.
+
+## Data gathering
+
+Data gathering is provided by the `grabdata.sh` script. When run, this script will
+execute a set of tests from the `tests/metrics` directory. The resulting JSON results files
+will be placed into the `tests/metrics/results` directory.
+
+Once the results have been generated, create a suitably named subdirectory of
+`tests/metrics/results`, and move the JSON files into it.
+
+Repeat this process if you wish to compare multiple sets of results. Note, the
+report generation scripts will process all subfolders of `tests/metrics/results' when
+generating the report.
+
+## Report generation
+
+Report generation is provided by the `makereport.sh` script. By default this script will
+process all subdirs of the tests/metrics/results directory to generate the report. To run
+in the default mode, execute:
+
+```sh
+$ ./makereport.sh
+```
+
+The report generation tool uses [Rmarkdown](https://github.com/rstudio/rmarkdown),
+[R](https://www.r-project.org/about.html) and [pandoc](https://pandoc.org/) to produce
+a PDF report. To save all users having to set up a working environment with all
+the necessary tooling, the `makereport.sh` script utilises a `Dockerfile` with
+the environment pre-defined in order to produce the report. Thus, you will need to
+have Docker installed on your system in order to run the report generation.
+
+The resulting `metrics_report.pdf` is generated into the `output` subdir of the `report`
+directory.
+
+# TODO
```

Comment:
I'd remove this and just raise a bunch of issues.

Reply:
Sure, can do. I'll open a single high level Issue with the tick list in that we can then use to spawn off individual issues on demand to fix the sub-items.